src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -498,16 +504,17 @@ export class MgtPersonCard extends MgtTemplatedComponent {
   }
 
   /**
-   * Use the sip: protocol to initiate a chat message to the user.
+   * Initiate a chat message to the user.
    *
    * @protected
    * @memberof MgtPersonCard
    */
   protected chatUser() {
     const user = this.personDetails as MicrosoftGraph.User;
-    if (user && user.mailNickname) {
-      const chat = user.mailNickname;
-      window.open('sip:' + chat, '_blank');
+    if (user && user.userPrincipalName) {
+      const users: string = [user.userPrincipalName].join(',');
```

Comment:
Why the join here?

Reply:
The api expects an array, and I just blindly did it this way. I'll fix it